CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ feel free to ask us and community.
 * fixed json/jsonb column data types comparison ([#3496](https://github.com/typeorm/typeorm/issues/3496))
 * fixed increment/decrement value of embedded entity ([#3182](https://github.com/typeorm/typeorm/issues/3182))
 * fixed missing call `transformer.from()` in case column is NULL ([#3395](https://github.com/typeorm/typeorm/issues/3395))
+* fixed mongodb entity listeners in optional embeddeds ([#3450](https://github.com/typeorm/typeorm/issues/3450))
```

Comment:
we need to move this changelog to the top now since that version was already released. Adding @vlapo - this can be problematic to maintain changelog properly since I guess there are other PRs with the same problem here

Reply:
@pleerock Probably we should use milestones for PRs and issues